lib/api/efd.js

```diff
@@ -2,7 +2,7 @@ import useSWR from "swr";
 
 const fetcher = (url) => fetch(url).then((res) => res.json());
 
-export default async function useEfd(type) {
```

Comment:
useSWR is doing the promisey stuff for us now so `useEfd` should no longer be `async`